public/js/blocks.js

```diff
@@ -301,10 +301,9 @@ class Edit extends Component {
           src: "https://cdn.razorpay.com/static/widget/subscription-button.js",
           "data-plugin": "wordpress_"+razorpay.payment_buttons_plugin_version,
           "data-subscription_button_id": buttonId,
```

Comment:
@neera11 why you had removed this? Any issue ?

Reply:
by default passing button theme , was creating problem